### ISSUE TITLE
Build tool remembers "mirror" state between buildings

### ISSUE THREAD

**Issue by dhouck** [ldtteam/Structurize#389]
### Prerequisites 

- [x] I am running the latest **alpha** version of Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.5
- Structurize Version: 0.13.177-alpha
- MineColonies Version (if related bug): 0.14.210-alpha
- Related Mods and their Versions: n/a

## Expected behavior

When you end a build tool session (green checkmark, red x, or either creative mode paste) the build tool forgets whatever the mirror setting was on the next use, like with rotation and positioning.

## Actual behavior

Mirror is remembered, leading to other structures being mirrored (potentially triggering bugs like https://github.com/ldtteam/minecolonies/issues/5006#issuecomment-856323324 or #377)

## Steps to reproduce the issue

1. Start placing a structure and choose to mirror it
2. Either finish placing the structure, or cancel the placement
3. Start placing another structure where you can tell if it's mirrored

### Logs

Logs do not show this issue

### Notes
I bet this is easy to fix but I haven't looked at the build tool code and I need to go to bed tonight instead of fixing it.

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Nightenom** [comment]
I don't see relations to either of linked issues
Also we don't reset mirror but also we don't reset rotation (along with few other values)

**Comment by dhouck** [comment]
Rotation is reset; I checked just now.  I started placing a building, rotated it 90°, hit the red X, and repeated, and it always started in the same rotation.

And the two linked issues are about ways mirroring doesn't work perfectly, so starting already mirrored can cause issues if you don't notice.  They don't cause this, and only one was caused by it, but they're a way that having things mirrored without realizing it can cause issues.
